libs/feature/search/src/lib/favorites/favorite-star/favorite-star.component.ts

```diff
@@ -74,10 +75,34 @@ export class FavoriteStarComponent implements AfterViewInit, OnDestroy {
         })
       }
     })
+    this.countSubscription = this.favoritesService.myFavoritesUuid$
+      .pipe(pairwise())
+      .subscribe((fav) => {
+        const oldFavs = fav[0]
+        const newFavs = fav[1]
+        let editedDs = []
+        if (oldFavs.length < newFavs.length) {
+          editedDs.push(newFavs.slice(-1)[0])
```

Comment:
We either have to choose if the modified uuids is a `string` or an `array`.
Here `editedDs` is an array but with at most one element, which is confusing.
The datahub does only need it to be a `string`, because we can change the selection just one by one.

Reply:
Okay! Will change it to a string then. It's just that I thought I remembered you or Tobias saying that in some cases, multiple favorites can be added at the same time and so I figured that an array would be best.